sdk/eventhub/azure-eventhub/azure/eventhub/_client_base.py

```diff
@@ -274,8 +282,16 @@ def get_token(self, *scopes, **kwargs):  # pylint:disable=unused-argument
 
 
 class ClientBase(object):  # pylint:disable=too-many-instance-attributes
-    def __init__(self, fully_qualified_namespace, eventhub_name, credential, **kwargs):
-        # type: (str, str, CredentialTypes, Any) -> None
+    def __init__(
+        self,
+        fully_qualified_namespace: str,
+        eventhub_name: str,
+        credential: CredentialTypes,
+        **kwargs: Any,
+    ) -> None:
+        self._uamqp_transport = kwargs.pop("uamqp_transport", True)
+        self._amqp_transport = kwargs.pop("amqp_transport", UamqpTransport)
```

Comment:
No so this uamqp_transport is a bool. By default, it will be False --> amqp_transport = PyamqpTransport. If users pass in True --> amqp_transport = UamqpTransport.

Reply:
We don't anymore, will remove it! (Originally, I was using it to build either the pyamqp-based EHSharedKeyCredential/etc. vs. uamqp-based EHSharedKeyCredential/etc. Missed removing it once we decided on just using pyamqp create_sas_token for the credentials, even with uamqp.)